VERSION

```diff
@@ -1 +1 @@
-0.1.0
+0.1.1
```

Comment:
Do we need to bump the version? I feel like keep it at 0.1.0 until we're ready to actually use it